<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFriendDlg</class>
 <widget class="QDialog" name="CFriendDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_headPic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_frdName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自我描述:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>141</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_sex"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_age"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_mail"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_address"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_cellphone"/>
      </item>
      <item>
       <widget class="QLineEdit" name="le_officephone"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>72</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>性别:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>年龄:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>邮箱:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>住址:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>移动电话:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>办公室电话:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="le_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_min">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lb_top</zorder>
  <zorder>lb_headPic</zorder>
  <zorder>lb_frdName</zorder>
  <zorder>label_7</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>le_description</zorder>
  <zorder>pb_close</zorder>
  <zorder>pb_min</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
